lib/features/login/presentation/screen/login_screen.dart

```diff
@@ -45,85 +49,140 @@ class _LoginScreenState extends State<LoginScreen> {
-              ZOClickableText(
-                  clickableText: context.l10n!.forgotPassword,
-                  color: ZOColors.onPrimaryLight,
-                  underline: false,
-                  onTap: () {
-                    context.router.push(const ForgotPasswordRoute());
-                  }),
-              const SizedBox(height: 30),
-            ],
+            ),
+          ],
+        );
+      },
+      mobile: (context) => _loginScreenContent(showImageInForm: true),
+    );
+  }
+
+  /// Builds the content of the login screen.
+  ///
+  /// The [showImageInForm] flag determines whether to show the image
+  /// in the form.
+  /// The [padding] parameter specifies the padding to add around the content.
+  Widget _loginScreenContent({
+    required bool showImageInForm,
+    EdgeInsetsGeometry? padding,
+  }) {
+    return SingleChildScrollView(
+      padding: padding,
+      child: Column(
+        children: <Widget>[
+          const SizedBox(height: GapSize.xl),
+          SvgPicture.asset(ZOStrings.zoLogoPath, width: 270, height: 46),
+          _formImageContent(showImageInForm),
+          Padding(
+            padding: const EdgeInsets.symmetric(
+              horizontal: WidgetStyle.padding,
+            ),
+            child: Form(
+              key: _formKey,
+              child: Column(
+                children: <Widget>[
+                  ZOTextField(
+                    label: context.l10n!.emailAddress,
+                    inputType: TextInputType.emailAddress,
+                    disableAutocorrect: true,
+                    controller: _emailController,
+                    onValidation: FieldValidationUtils.getEmailValidator(
+                      context,
+                    ),
+                  ),
+                  const SizedBox(height: GapSize.m),
+                  ZOPasswordTextField(
+                    text: context.l10n!.password,
+                    controller: _passwordController,
+                    // Add more bottom padding to scroll above a snackbar
+                    scrollPadding: const EdgeInsets.only(
+                      left: 20,
+                      top: 20,
+                      right: 20,
+                      bottom: 128,
+                    ),
+                    onValidation: FieldValidationUtils.getPasswordValidator(
+                      context,
+                    ),
+                  ),
+                  const SizedBox(height: GapSize.l),
+                  ZOButton(
+                    text: context.l10n!.signIn,
+                    icon: MaterialSymbols.login,
+                    onPressed: () async {
+                      if (_formKey.currentState!.validate()) {
+                        await _logIn();
+                      }
+                    },
+                  ),
+                ],
+              ),
+            ),
           ),
-        ),
+          const SizedBox(height: GapSize.xs),
+          ZOClickableText(
```

Comment:
Nit: Would not it be better to use a ZOButton with some modification (a new type, it will have a transparent background) to make it look like a clickable (background and cursor changed)? As it is, this appears unclickable on the web.

Reply:
Good point, I have added new types: `tertiary` for login (with dark text color) and `cancel` for offer donation form (with red text color) 👍